pages/api/stats/index.ts

```diff
@@ -170,9 +172,13 @@ export default withAuth({
           }], { session: session });
 
           if (complete) {
+            const discordTxt = `Congrats you beat x amount of levels`
```

Comment:
Why do anything with discord for achievements? We would have to figure out how to private message them right? Otherwise this would go into public channels

Reply:
I was having difficulty figuring out how to get `Congrats you beat x amount of levels` message into the notification queue and tried recreating what was done on a notification I could produce by reviewing a level, I believe.